vnext/Shared/Networking/IRedirectEventSource.idl

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.React.Networking {
+
+  interface IRedirectEventSource
+  {
+  Boolean OnRedirecting(
+        Windows.Web.Http.HttpRequestMessage request,
+        Windows.Web.Http.HttpResponseMessage response);
```

Comment:
Please document WinRT types and APIs (see DOC_STRING in the other IDLs)

Reply:
"DocString.h" was not found (this type is defined in the Shared items project, and is not meant to be part of the public API or the DLL boundary).

I have added doc comments.